google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -239,26 +274,42 @@ private void recordOperationCompletion(@Nullable Throwable status) {
       return;
     }
     operationTimer.stop();
-    long operationLatency = operationTimer.elapsed(TimeUnit.MILLISECONDS);
+
+    boolean isStreaming = operationType == OperationType.ServerStreaming;
+    String statusStr = Util.extractStatus(status);
+
+    Attributes attributes =
+        baseAttributes
+            .toBuilder()
+            .put(TABLE_ID_KEY, tableId)
+            .put(CLUSTER_ID_KEY, cluster)
+            .put(ZONE_ID_KEY, zone)
+            .put(METHOD_KEY, spanName.toString())
+            .put(CLIENT_NAME_KEY, NAME)
+            .build();
```

Comment:
question, can you just put all of the attributes here (ie add STATUS_KEY and IS_STREAMING)? I think your views should just ignore the attributes they dont care about and it will clean up all of the `put(STATUS_KEY)` invocations you have sprinkled everywhere